lib/ukatomic/include/uk/atomic.h

```diff
@@ -27,83 +27,81 @@
  * SUCH DAMAGE.
  */
 
-#ifndef __UKARCH_ATOMIC_H__
-#define __UKARCH_ATOMIC_H__
+#ifndef __UK_ATOMIC_H__
+#define __UK_ATOMIC_H__
 
 #include <uk/arch/lcpu.h>
 
 #ifdef __cplusplus
 extern "C" {
```

Comment:
I'm leaving this here as github doesn't allow to comment on the closing brace: I think there is no reason to have `ACCESS_ONCE` and `UK_READ_ONCE` / `UK_WRITE_ONCE` outside the C++ guards. I checked on the commit message that introduced these changes and didn't see any specific reason mentioned, so I speculate it was an omission.